### PR TITLE
Add "plain" mode (disables mounts, port forwarding, containerd, etc.)

### PR DESCRIPTION
**Pull request by AkihiroSuda**
```bash
limactl start --plain
```

When the "plain" mode is enabled:
- the YAML properties for mounts, port forwarding, containerd, etc. will be ignored
- guest agent will not be running.
- dependency packages like sshfs will not be installed into the VM

User-specified provisioning scripts will be still executed.

Fixes:
- #1739

### CODE REVIEW

File: pkg/start/start.go [hunk 1]
```diff
@@ -240,7 +240,11 @@ func watchHostAgentEvents(ctx context.Context, inst *store.Instance, haStdoutPat
 				return true
 			}
 
-			logrus.Infof("READY. Run `%s` to open the shell.", LimactlShellCmd(inst.Name))
+			if *inst.Config.Plain {
+				logrus.Infof("READY. Run `ssh -F %q lima-%s` to open the shell.", inst.SSHConfigFile, inst.Name)
```
Comment (balajiv113) [inline pkg/start/start.go:244]: Y not use LimactlShellCmd(inst.Name) here ?? Ssh connection stays the same right evn for plan mode ?

Comment (AkihiroSuda) [reply pkg/start/start.go:244]: Because plain mode users are less likely to prefer lima-ist commands.
The limactl shell command continue to work.

**Review by balajiv113 — APPROVED** [review]

File: pkg/start/start.go (hunk 1 above)
Comment (afbjorklund) [reply pkg/start/start.go:244]: I still think the `lima` wrapper (but only that script, not the others) belongs to the "minimal" footprint.

* https://github.com/lima-vm/lima/pull/1673

But I suppose `ssh` is even more "plain".

**Comment by afbjorklund** [comment]
Here was the suggestion for a "minimal" version of the regular `lima` command, wrapping `ssh` directly instead.

```sh
#!/bin/sh
exec ssh -F "${LIMA_HOME:-$HOME/.lima}/$(echo $1 | sed s/lima-//)/ssh.config" $LIMA_INSTANCE "$@"
```

What do you think ? It is just to make the invocation more "pretty", by hiding the path to the ssh.config location...

```
INFO[0020] READY. Run `ssh -F "/home/anders/.lima/default/ssh.config" lima-default` to open the shell. 
```

----

READY. Run `lima-ssh lima-default`

**Pull request merged**